unit-tests/live/image-quality/test-image-quality.py

```diff
@@ -0,0 +1,155 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+
+import pyrealsense2 as rs
+from rspy.stopwatch import Stopwatch
+from rspy import test, log
+import time
+import sys
+import os
+
+# Start depth + color streams and go through the frame to make sure it is showing a depth image
+# Color stream is only used to display the way the camera is facing
+# Verify that the frame does indeed have variance - therefore it is showing a depth image
+
+dev = test.find_first_device_or_exit()
+
+cfg = rs.config()
+cfg.enable_stream(rs.stream.depth, rs.format.z16, 30)
+cfg.enable_stream(rs.stream.color, rs.format.rgb8, 30)
+
+
+def display_image(img):
+    """
+    Display a given image and exits when the x button or esc key are pressed
+    """
+    import cv2
+
+    window_title = "Output Stream"
+    cv2.imshow(window_title, img)
+    while cv2.getWindowProperty(window_title, cv2.WND_PROP_VISIBLE) > 0:
+        k = cv2.waitKey(33)
+        if k == 27:  # Esc key to stop
+            cv2.destroyAllWindows()
+            break
+        elif k == -1:  # normally -1 returned,so don't print it
+            pass
+
+
+def frames_to_image(depth, color, save, display):
+    import numpy as np
+    import cv2
+
+    colorizer = rs.colorizer()
+    depth_image = np.asanyarray(colorizer.colorize(depth).get_data())
+    img = depth_image
+
+    if color:  # if color frame was successfully captured, merge it and the depth frame
+        from scipy.ndimage import zoom
+        color_image = np.asanyarray(color.get_data())
+        depth_rows, _, _ = depth_image.shape
+        color_rows, _, _ = color_image.shape
+        # resize the image with the higher resolution to look like the smaller one
+        if depth_rows < color_rows:
+            color_image = zoom(color_image, (depth_rows / color_rows, depth_rows / color_rows, 1))
+        elif color_rows < depth_rows:
+            depth_image = zoom(depth_image, (color_rows / depth_rows, color_rows / depth_rows, 1))
+        img = np.concatenate((depth_image, color_image), axis=1)
+
+    if save:
+        file_name = "output_stream.png"
+        print("Saved image in", os.getcwd() + "\\" + file_name)
```

Comment:
Don't `print` use the `log` mechanism you have imported.
1. There isn't always a screen, log messages are kept in a file if needed.
2. You can set the verbosity level f/e/i/d (fatal don't continue / error but try to continue / info message / debug message only)